lib/helpers/Profile/profileActiveHelpers.js

```diff
@@ -1,9 +1,12 @@
 if (Meteor.isClient){
 	Template.ProfileActive.helpers({
 		// TODO: Select Accepted listings that are of the users own id or have the acceptedOffer of the users offer
+		
 		meetup: function () {
+			var offerid = this.offerAccepted;
+
 			var id = Meteor.userId();
-			return Listing.find({ creator_id: Meteor.userId(), status: "Accepted" });
+			return Listing.find({ $and:[ { creator_id: Meteor.userId(), status: "Accepted" }, { offerAccepted: Offer.find({_id: this.offerAccepted})  } ] });
```

Comment:
Line is too long.<br>Identifier 'creator_id' is not in camel case.<br>'Listing' is not defined.<br>'Meteor' is not defined.<br>'Offer' is not defined.